modules/core/src/main/scala/lucuma/odb/api/schema/TargetMutation.scala

```diff
@@ -164,14 +164,36 @@ trait TargetMutation extends TargetScalars {
       "Sidereal target description"
     )
 
-  val InputObjectTypeTargetEditSidereal: InputObjectType[TargetModel.EditSidereal] =
+  implicit val InputObjectTypeTargetCreate: InputObjectType[TargetModel.Create] =
+    deriveInputObjectType[TargetModel.Create](
+      InputObjectTypeName("CreateTargetInput"),
+      InputObjectTypeDescription("Target creation parameters")
+    )
+
+  val ArgumentTargetCreate: Argument[TargetModel.Create] =
+    InputObjectTypeTargetCreate.argument(
+      "input",
+      "Target creation parameters.  Choose 'nonSidereal' or 'sidereal'."
+    )
+
+
+  implicit val InputObjectTypeEditNonsidereal: InputObjectType[TargetModel.EditNonsidereal] =
+    deriveInputObjectType[TargetModel.EditNonsidereal](
+      InputObjectTypeName("EditNonsiderealInput"),
+      InputObjectTypeDescription("Nonsidereal target edit parameters"),
+
+      ReplaceInputField("name", StringType      .notNullableField("name")),
```

Comment:
Should this be a `NonEmptyString`?

Reply:
Yes thanks, I believe you're right.  It should be the `NonEmptyStringType` scalar.